### ISSUE TITLE
Fout 400: invalid_request - urn:ietf:wg:oauth:2.0:oob

### ISSUE THREAD

**Issue by Zaurio20** [viperadnan-git/google-drive-telegram-bot#112]
Hello. i'm getting this error when trying to give access to my drive account.

The out-of-band (OOB) flow has been blocked in order to keep users secure. Follow the Out-of-Band (OOB) flow Migration Guide linked in the developer docs below to migrate your app to an alternative method.
Details van verzoek: redirect_uri=urn:ietf:wg:oauth:2.0:oob

Is this bot still working?

**Comment by 0x656d707479** [comment]
I have the same problem right now. And I do not know how to fix it. If you decide to explain yourself, please.

**Comment by Zaurio20** [comment]
> I have the same problem right now. And I do not know how to fix it. If you decide to explain yourself, please.

Can't solve it. I tried by changing REDIRECT_URI from the authorize.py file but didn't make it work.

This one is still working if you need: https://github.com/zume2020/Telegram-bot-Google-Drive

Since telegram only allow up to 20 MB per file to upload, I think none of both bots are what I need.

**Comment by 0x656d707479** [comment]
> Can't solve it. I tried by changing REDIRECT_URI from the authorize.py file but didn't make it work.
> This one is still working if you need: https://github.com/zume2020/Telegram-bot-Google-Drive
> Since telegram only allow up to 20 MB per file to upload, I think none of both bots are what I need.

Yes, I also tried to change something there, but nothing worked for me either.  My friend said that the problem there is in redirection.

Thanks for the link to the second bot.

If I find a solution to the problem (Or if a friend helps me fix this error) then I will write how to fix it.

**Comment by wx69wx** [comment]
there is the reason.  google think the oauth method (oob)  is unsafe.   so it close the support .the bot must use the new oauth method.

https://developers.google.com/identity/protocols/oauth2/resources/oob-migration

**Comment by some60dy** [comment]
Who has been able to go out of band (OOB) and set up authentication? Please, help. I really need this bot!

**Comment by some60dy** [comment]
Who has been able to bypass the OOB block and set up authentication? Please, help. I really need this bot!

**Comment by thethor12** [comment]
> > I have the same problem right now. And I do not know how to fix it. If you decide to explain yourself, please.
> 
> Can't solve it. I tried by changing REDIRECT_URI from the authorize.py file but didn't make it work.
> 
> This one is still working if you need: https://github.com/zume2020/Telegram-bot-Google-Drive
> 
> Since telegram only allow up to 20 MB per file to upload, I think none of both bots are what I need.

That zume2020 is not asking for client IDs and secrets and it's not getting deployed

**Comment by khainee** [comment]
I know how to make repo alive

**Comment by BotTutorials** [comment]
> Hello. i'm getting this error when trying to give access to my drive account.
> 
> The out-of-band (OOB) flow has been blocked in order to keep users secure. Follow the Out-of-Band (OOB) flow Migration Guide linked in the developer docs below to migrate your app to an alternative method. Details van verzoek: redirect_uri=urn:ietf:wg:oauth:2.0:oob
> 
> Is this bot still working?

#118

**Comment by BotTutorials** [comment]
#118
